distribution/lib/Standard/Base/0.0.0-dev/src/Data/Filter_Condition.enso

```diff
@@ -105,6 +105,12 @@ type Filter_Condition
          See https://bugs.java.com/bugdatabase/view_bug.do?bug_id=8032926
     Not_Like pattern:Text
 
+    ## Is the value contained in `values`?
+    Is_In values:Vector
+
+    ## Is the value not contained in `values`?
+    Not_In values:Vector
```

Comment:
@jdunkerley do you think we should have a comment saying that this will not be as efficient for mixed type columns because of lack of hashing?
Or do we keep it as an implementation detail (still the user may notice that depending on column type it's either pretty fast or very slow).